control-plane-operator/controllers/hostedcontrolplane/storage/assets/10_deployment-hypershift.yaml

```diff
@@ -19,6 +19,8 @@ spec:
       - args:
         - start
         - -v=2
+        - --terminate-on-files=/var/run/secrets/serving-cert/tls.crt
```

Comment:
I don't see where this is mounted into the pod

Reply:
Thanks for nice catch, @csrwng ! It is mounted in standalone version of deployment (https://github.com/openshift/cluster-storage-operator/blob/master/manifests/10_deployment.yaml#L151) but not in hypershift. I'll close this PR as erroneous.